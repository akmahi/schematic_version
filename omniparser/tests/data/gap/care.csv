"Row","Domain","Patient First Name","Patient Last Name","Patient DOB","External Patient ID","External Practice ID","Code","Name","Dictionary Name","Date Performed","Date Ordered","Date Due","Refused","Med Unnecessary RSN Code","Med Unnecessary RSN Name","Med Unnecessary RSN Dict","Visit Reason Code","Visit Reason Name","Visit RSN Dict Name","NPI","TIN","Provider Name","Remove Care Item","Value","Modifier Dictionary Name","Modifier Code","Vendor","Source Type","PATIENT_ID"
"1","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: DEPRESSION","","20220422","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"2","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: DEPRESSION","","20210428","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"3","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: DEPRESSION","","20220923","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"4","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: FALL RISK","","20190319","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"5","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: FALL RISK","","20200303","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"6","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: FALL RISK","","20210416","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"7","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: FALL RISK","","20220517","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"8","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","SCREENING: GONORRHEA","","20220520","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"9","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","BMI Plan","","20220517","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
"10","Cares","SIMONE","ABRAMS","19641214","3368664","Epic","","Cervical Cancer Screening","","20220520","","","","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V"
