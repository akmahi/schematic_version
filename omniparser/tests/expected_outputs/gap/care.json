[
	{
		"care": {
			"date_performed": "20220422",
			"domain": "Cares",
			"name": "SCREENING: DEPRESSION",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "1"
		}
	},
	{
		"care": {
			"date_performed": "20210428",
			"domain": "Cares",
			"name": "SCREENING: DEPRESSION",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "2"
		}
	},
	{
		"care": {
			"date_performed": "20220923",
			"domain": "Cares",
			"name": "SCREENING: DEPRESSION",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "3"
		}
	},
	{
		"care": {
			"date_performed": "20190319",
			"domain": "Cares",
			"name": "SCREENING: FALL RISK",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "4"
		}
	},
	{
		"care": {
			"date_performed": "20200303",
			"domain": "Cares",
			"name": "SCREENING: FALL RISK",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "5"
		}
	},
	{
		"care": {
			"date_performed": "20210416",
			"domain": "Cares",
			"name": "SCREENING: FALL RISK",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "6"
		}
	},
	{
		"care": {
			"date_performed": "20220517",
			"domain": "Cares",
			"name": "SCREENING: FALL RISK",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "7"
		}
	},
	{
		"care": {
			"date_performed": "20220520",
			"domain": "Cares",
			"name": "SCREENING: GONORRHEA",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "8"
		}
	},
	{
		"care": {
			"date_performed": "20220517",
			"domain": "Cares",
			"name": "BMI Plan",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "9"
		}
	},
	{
		"care": {
			"date_performed": "20220520",
			"domain": "Cares",
			"name": "Cervical Cancer Screening",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "10"
		}
	}
]
